data/org.freedesktop.portal.Clipboard.xml

```diff
@@ -0,0 +1,180 @@
+<?xml version="1.0"?>
+<!--
+ Copyright 2022 Google LLC
+
+ This library is free software; you can redistribute it and/or
+ modify it under the terms of the GNU Lesser General Public
+ License as published by the Free Software Foundation; either
+ version 2 of the License, or (at your option) any later version.
+
+ This library is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ Lesser General Public License for more details.
+
+ You should have received a copy of the GNU Lesser General Public
+ License along with this library. If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<node name="/"
+  xmlns:doc="http://www.freedesktop.org/dbus/1.0/doc.dtd">
+  <!--
+      org.freedesktop.portal.Clipboard
+      @short_description: Clipboard portal
+
+      The Clipboard portal allows sessions to access the clipboard.
+
+      This documentation describes version 1 of this interface.
+  -->
+  <interface name="org.freedesktop.portal.Clipboard">
+    <!-- 
+        RequestClipboard:
+        @session_handle: Object path for the #org.freedesktop.portal.Session object
+        @options: Vardict with optional further information
+
+        Requests clipboard access for the given portal session. This request
+        must be made before the session starts.
+
+        This portal does NOT create it's own session. Instead, it offers existing sessions
+        created from other portals the option to integrate with the clipboard. For whether
+        this interface is supported for a given session, refer to that portal's documentation.
+        See #org.freedesktop.portal.RemoteDesktop to integrate clipboard with the 
+        remote desktop session.
+    -->
+    <method name="RequestClipboard">
+      <arg type="o" name="session_handle" direction="in"/>
+      <arg type="a{sv}" name="options" direction="in"/>
+    </method>
+    <!-- 
+        SetSelection:
+        @session_handle: Object path for the #org.freedesktop.portal.Session object
+        @options: Vardict with optional further information
+
+        Sets the owner of the clipboard formats in 'mime_types' in @options to
+        the session, i.e. this session has data for the advertised clipboard formats.
```

Comment:
Could be good to link to `org.freedesktop.portal.FileTransfer` to point out when to use the `application/vnd.portal.filetransfer` mimetype, e.g. when transferring files via otherwise done via file lists.